monthly-contributions/src/main/scala/com/gu/support/workers/lambdas/CreatePaymentMethod.scala

```diff
@@ -50,8 +50,9 @@ class CreatePaymentMethod(servicesProvider: ServiceProvider = ServiceProvider)
       .createCustomer(stripe.userId, stripe.stripeToken)
       .map { stripeCustomer =>
         val card = stripeCustomer.card
+        val zuoraCardType = card.`type`.replaceAll(" ", "") // Zuora requires 'AmericanExpress' not 'American Express'
```

Comment:
Could maybe move this into the Stripe.Card class as there is another derived value in there already. I don't feel super strongly about it though if you'd rather keep it here.

Reply:
I'm assuming you've checked that there are no types for which we'd want to retain spaces?

Reply:
Good idea - I've also linked to the docs which lists the types that Zuora will accept